nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-components/src/test/java/org/apache/nifi/parameter/TestStandardParameterContext.java

```diff
@@ -331,17 +332,17 @@ public void testChangingNestedParameterForRunningProcessor() {
         startProcessor(procNode);
         try {
             a.setInheritedParameterContexts(Collections.emptyList());
-            Assert.fail("Was able to remove parameter while referencing processor was running");
```

Comment:
This should be replaced with something like `IllegalStateException ex = assertThrows(...`